.github/workflows/ruby.yml

```diff
@@ -36,15 +36,24 @@ jobs:
     - name: Setup Node
       uses: actions/setup-node@v1
       with:
-        node-version: 12.14.0
+        node-version: 12.16.3
 
     - name: Install node deps
       run: |
         npm install -g yarn
         yarn install
+        sudo apt-get install libsqlite3-dev
 
     - name: Run tests
-      run: bundle exec rake
+      run: |
+        bundle exec rake
```

Comment:
This is running the tests twice, we either want `bundle exec rake` or the stanza below it